bluemira/equilibria/physics.py

```diff
@@ -455,3 +463,195 @@ def beta_N_to_beta(beta_N, a, Btor, Ip):  # noqa :N802
 
     """
     return beta_N * Ip / (a * Btor)
+
+
+def debye_length(temperature, density):
+    """
+    Debye length
+
+    Parameters
+    ----------
+    temperature: float
+        Temperature [K]
+    density: float
+        Density [m^-3]
+
+    Returns
+    -------
+    debye_length: float
+        Debye length [m]
+    """
+    return np.sqrt(EPS_0 * K_BOLTZMANN * temperature / (EV_TO_J**2 * density))
+
+
+def reduced_mass(mass_1, mass_2):
+    """
+    Calculate the reduced mass of a two-particle system
+
+    Parameters
+    ----------
+    mass_1: float
+        Mass of the first particle
+    mass_2: float
+        Mass of the second particle
+
+    Returns
+    -------
+    mu_12: float
+        Reduced mass
+    """
+    return (mass_1 * mass_2) / (mass_1 + mass_2)
+
+
+def thermal_velocity(temperature, mass):
+    """
+    Parameters
+    ----------
+    temperature: float
+        Temperature [K]
+    mass: float
+        Mass of the particle [kg]
+
+    Notes
+    -----
+    The sqrt(2) term is for a 3-dimensional system and the most probable velocity in
+    the particle velocity distribution.
+    """
+    return np.sqrt(2) * np.sqrt(K_BOLTZMANN * temperature / mass)
+
+
+def de_broglie_length(velocity, mu_12):
+    """
+    Calculate the de Broglie wavelength
+
+    Parameters
+    ----------
+    velocity: float
+        Velocity [m/s]
+    mu_12: float
+        Reduced mass [kg]
+
+    Returns
+    -------
+    lambda_de_broglie: float
+        De Broglie wavelength [m]
+    """
+    return H_PLANCK / (2 * mu_12 * velocity)
+
+
+def impact_parameter_perp(velocity, mu_12):
+    """
+    Calculate the perpendicular impact parameter
+
+    Parameters
+    ----------
+    velocity: float
+        Velocity [m/s]
+    mu_12: float
+        Reduced mass [kg]
+
+    Returns
+    -------
+    b90: float
+        Perpendicular impact parameter [m]
+    """
+    return EV_TO_J**2 / (4 * np.pi * EPS_0 * mu_12 * velocity**2)
+
+
+def coulomb_logarithm(temperature, density):
+    """
+    Calculate the value of the Coulomb logarithm for an electron hitting a proton.
+
+    Parameters
+    ----------
+    temperature: float
+        Temperature [K]
+    density: float
+        Density [1/m^3]
+
+    Returns
+    -------
+    ln_lambda: float
+        Coulomb logarithm value
+    """
+    lambda_debye = debye_length(temperature, density)
+    mu_12 = reduced_mass(ELECTRON_MASS, PROTON_MASS)
+    v = thermal_velocity(temperature, ELECTRON_MASS)
+    lambda_de_broglie = de_broglie_length(v, mu_12)
+    b_perp = impact_parameter_perp(v, mu_12)
+    b_min = max(lambda_de_broglie, b_perp)
+    return np.log(np.sqrt(1 + (lambda_debye / b_min) ** 2))
+
+
+def spitzer_conductivity(Z_eff, T_e, ln_lambda=17):
+    """
+    Formula for conductivity as per L. Spitzer
```

Comment:
Is there more detail we could give here, for those not familiar with Spitzer (me)? The conductivity through what?

From a quick google, is the below correct?

```suggestion
    Formula for electrical conductivity in a plasma as per L. Spitzer
```